src/Servant/QuickCheck/Internal/HasGenRequest.hs

```diff
@@ -195,3 +195,6 @@ instance (HasGenRequest a) => HasGenRequest (WithNamedContext x y a) where
 -- TODO: Try logging in
 instance (HasGenRequest a) => HasGenRequest (BasicAuth x y :> a) where
     genRequest _ = genRequest (Proxy :: Proxy a)
+
```

Comment:
`Fragment a` is a data slot.
I think that one should generate test data and include it.

Reply:
Fragments are ignored in `HasServer`/`HasClient` instances, see e.g. [*Proposal for Servant \#3* in this blog post](https://an-pro.org/posts/11-servant-fragment.html) for why this is the case:

```haskell
 Λ :kind! Server (Fragment String :> Get '[JSON] Int)
Server (Fragment String :> Get '[JSON] Int) :: *
= Handler Int
```
Also, `http-client` does not send fragments (like curl and web browsers):
```haskell
 Λ parseRequest "https://example.org/path#fragment"
Request {
  host                 = "example.org"
  port                 = 443
  secure               = True
  requestHeaders       = []
  path                 = "/path"
  queryString          = ""
  method               = "GET"
  proxy                = Nothing
  rawBody              = False
  redirectCount        = 10
  responseTimeout      = ResponseTimeoutDefault
  requestVersion       = HTTP/1.1
  proxySecureMode      = ProxySecureWithConnect
}
```